apps/wallet-connect/src/components/WCClientInfo.tsx

```diff
@@ -26,7 +26,7 @@ type Props = {
 const WCClientInfo = ({ name, url, iconSrc }: Props) => {
   return (
     <Wrapper>
-      <Image src={iconSrc} />
+      <Image src={iconSrc} role="img" />
```

Comment:
This is for Accessibility purposes, `ARIA img role` is used to identify this div element as an image.

see docs for more info: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role